doc/classes/ResourceFormatLoader.xml

```diff
@@ -17,6 +17,13 @@
 			<description>
 			</description>
 		</method>
+		<method name="_get_attached_script_path" qualifiers="virtual const">
+			<return type="String" />
+			<argument index="0" name="path" type="String" />
+			<description>
+				If implemented, gets the path of a script attached to a resource. Scripts are only attached to a resource for custom classes that inherits from [Resource].
```

Comment:
```suggestion
				If implemented, gets the path of a script attached to a resource. Scripts are only attached to a resource for custom classes that inherit from [Resource].
```
The second sentence sounds a bit confusing; could be reworded.

Reply:
Would this work?

If implemented, gets the path of a script attached to a resource. An attached script only exists for custom resources, which are user-defined classes that inherit from [Resource].

Reply:
Yes, this is more clear.
Although not really true, as you can attach a script to any Resource via the inspector.

Reply:
Should we add a bit for that then?

Reply:
I think something like
```suggestion
				If implemented, gets the path of a script attached to a resource. The script is usually attached when creating a new instance of a [Resource]-based custom class.
```
should be fine. Although that sentence could as well be removed, as it's not that important where the script comes from.